README.md

```diff
@@ -34,20 +34,23 @@ import { h } from 'preact';
 import {storiesOf} from '@storybook/preact';
 import {withAmp} from '@ampproject/storybook-addon';
 
-storiesOf('amp-carousel', module)
-  .addDecorator(withAmp)
-  .addParameters({
-    extensions: [{name: 'amp-carousel', version: '0.2'}],
-  })
-  .add('default', () => {
-    return (
-      <amp-base-carousel width="440" height="225">
-        {['lightcoral', 'peachpuff', 'lavender'].map((color) => (
-          <div key={color}>{color}</div>
-        ))}
-      </amp-base-carousel>
-    );
-  });
+export default {
+  title: 'amp-carousel',
+  decorators: [withAmp],
+  parameters: {
+    extensions: [{name: 'amp-carousel', version: '0.2'}]},
+  },
+};
+
+export const Default = () => {
```

Comment:
I think it's recommended to call the default one `_default`?

Reply:
🤷‍♂️

Could not find `_default` in their docs, which would contradict this:

> [**Named story exports**](https://storybook.js.org/docs/react/api/csf#named-story-exports)
> ...
> It's recommended to start export names with a capital letter.

But if we prefer `_default` I'd be happy to make the change. We can consider `_Default` too.